packages/data-point/lib/resolve/reducer-path.js

```diff
@@ -1,56 +1,16 @@
 'use strict'
 
-const _ = require('lodash')
 const utils = require('../utils')
 
 /**
- * resolve the JSON Object path from given value
- *
- * @param {Object} value
- * @param {string} jsonPath
- * @returns
- */
-function resolveObjectPath (acc, jsonPath, reducerPath) {
-  if (typeof reducerPath === 'undefined') {
-    reducerPath = {}
-  }
-
-  const type = reducerPath.type || ''
-  const asCollection = reducerPath.asCollection || false
-
-  if (jsonPath === '.' || _.isEmpty(jsonPath)) {
-    return acc.value
-  }
-
-  if (jsonPath.substring(0, 2) === '..') {
-    return _.get(acc, jsonPath.slice(2))
-  }
-
-  if (type === 'ReducerPath' && asCollection) {
-    if (!Array.isArray(acc.value)) {
-      return null
```

Comment:
this should had been undefined, my bad

Reply:
There's an open issue for addressing this https://github.com/ViacomInc/data-point/issues/60